sentry_sdk/integrations/spark/__init__.py

```diff
@@ -0,0 +1,4 @@
+from sentry_sdk.integrations.spark.spark_driver import SparkIntegration
+from sentry_sdk.integrations.spark.spark_worker import sentry_worker_main
+
+__all__ = ["SparkIntegration", "sentry_worker_main"]
```

Comment:
Why do we need to export `sentry_worker_main` for users to patch in? Why can't we patch `main` ourselves?

Reply:
We can't patch main ourselves because the code has already been sent to the distributed executors by the time we patch `SparkContext` (which is the earliest we can do anything). As a result, we opt to set it as a spark config option and have the users create their own file. 

I know this means extra steps for our users, I am actively looking at alternate methods.

Reply:
Do you think we should also have a `SparkWorkerIntegration` in this case?